test/WebSites/BasicWebSite/Controllers/ContactApiController.cs

```diff
@@ -72,5 +73,27 @@ public ActionResult<Contact> ActionWithInferredEmptyPrefix([FromQuery] Contact c
         {
             return contact;
         }
+
+        [HttpGet("[action]")]
+        public ActionResult<string> ActionWithInferredModelBinderType(
+            [ModelBinder(typeof(TestModelBinder), Name = "bar")] string foo)
```

Comment:
Could you add one more scenario without the explicit binder name? It involves interactions with `ParameterBinder` so we'd like to make sure it works end-to-end.